### DISCUSSION TITLE
Trouble with making a directory and subdirectories "vendored"

### DISCUSSION THREAD

**Discussion by AbsoluteWisp** [github-linguist/linguist#5491] [Classification] [answered]
I have a repository ([link](https://github.com/Ghostling225/TopDownShooter/)), where I would like the `**/TextMesh Pro/**` (all files and subfolders in any folder called "TextMesh Pro") filter to be considered "vendored" by linguist. I tried adding this line to `.gitattributes` in the repo root:  
`"**/TextMesh Pro/**" linguist-vendored`, but it's still not working (my code is exclusively C#, but linguist still shows HLSL and ShaderLab as the vast majority of the repo's code. What am I doing wrong?

**Comment by lildude** [comment, marked as answer]
You'll need to escape the space with a backslash: `**/TextMesh\ Pro/** linguist-vendored` or use `[[:space:]]`: `**/TextMesh[[:space:]]Pro/** linguist-vendored` 

Either should do the trick.

**Reply by AbsoluteWisp** [reply]
Thank you! I'm new to both git and GitHub and thought enclosing it in a string would magically make the space not a problem. Git gave me an error when I tried escaping the space with a backslash, but the `[[:space:]]` tag worked perfectly! Now linguist sees only the C# that is actually mine. Thanks again!
